cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -2411,12 +2429,19 @@ def get_xblock_problem(label):
             problem_templates = self.get_templates_of_type('problem')
             return self.get_template(problem_templates, label)
 
-        def verify_openassessment_present(support_level):
+        def verify_openassessment_not_present():
             """ Helper method to verify that openassessment template is present """
             openassessment = get_xblock_problem('Open Response Assessment')
-            self.assertIsNotNone(openassessment)
-            self.assertEqual(openassessment.get('category'), 'openassessment')
-            self.assertEqual(openassessment.get('support_level'), support_level)
+            self.assertIsNone(openassessment)
+
+        def verify_staffgradedxblock_present(support_level):
```

Comment:
Why is this being added? Same as the previous comment.

Reply:
This is to check that there are multiple component loaded under `problems` block , just like before.

Reply:
Ah, good to keep then :)